esp/scm/ws_topology.ecm

```diff
@@ -17,6 +17,13 @@
 
 ////////////////////////////////////////////////////////////
 
+ESPenum DeployOnlyFilter : string
+{
+    All("All"),
+    DeployOnly("DeployOnly"),
+    NotDeployOnly("NotDeployOnly")
+};
```

Comment:
@richardkchapman Just want to make sure we have the intention and terminology right here.
In many places deploy some ecl is to compile the workunit.. that's not the same thing as publishing a query.  So there are 3 actions.
1. deploy / compile a workunit.
2. run a workunit
3. publish a query.
ECL could be compiled for the roxie (deployed as workunits) but wouldn't end up callable as queries yet.

If the focus is on the workunit it could be:
All, Runnable, CompileOnly.

If the focus is on how things are run it could be:
All, WorkunitsOnly, QueriesOnly

In both cases we really allow publishing/compiling to other cluster types so in practice it would just be All and either CompileOnly or QueriesOnly.

Reply:
I guess we are trying to separate 2 groups of roxies here: the roxies for running (compile and run) workunits and the roxies for publishing queries. If so, we should change to 'All, WorkunitsOnly, QueriesOnly'. But, I could be wrong.

Reply:
Makes sense to me.